app/views/staff/reports/activities.html.haml

```diff
@@ -15,18 +15,7 @@
         .govuk-tabs__panel
           %h2.govuk-heading-l
             = t("tabs.report.activities.heading")
-
-          - if policy(@report_presenter).upload?
```

Comment:
I noticed that the conditional has disappeared in the move to the partial. Is this intentional?

Reply:
Whoops, no. Fixed.